packit_service/worker/helpers/build/copr_build.py

```diff
@@ -493,6 +539,7 @@ def submit_copr_build(self, script: Optional[str] = None) -> Tuple[int, str]:
                     buildopts={
                         "chroots": list(self.build_targets),
                         "enable_net": self.job_config.enable_net,
+                        "packit_forge_project": self.forge_project,
```

Comment:
I realised we should probably include this in the dict (also in the other place) only if we are handling with custom Copr project and if that project is not configured in Packit (the second condition should be removed when we get rid of the configuration on Packit side)

Reply:
You are right!